index.js

```diff
@@ -96,6 +101,10 @@ module.exports = class Combobox {
       this.freshSelection = true;
     });
 
+    this.input.addEventListener('focus', () => {
+      this.input.value = this.selected >= 2 ? this.config.selectionValue(this.selected) : '';
```

Comment:
this should be `this.selected.length`. Also, correct me if I'm wrong, but I thought we wanted to clear the input's value on focus of the input when multiple items are selected?